src/components/Chart/index.js

```diff
@@ -46,7 +46,10 @@ export class Chart extends Component {
     }
 
     renderChart() {
-        const { type, labels, ...conditions } = this.props;
+        const { type, labels, plugins, ...conditions } = this.props;
+        if (plugins) {
+            ChartJS.plugins.unregister(plugins);
```

Comment:
why we have to unregister the plugins?? I know the labels one needs to be unregistered ... but I guess that's not the case for all the plugins, right?

Again this feature should a mechanism to inject any possible chart plugin

Reply:
I was looking at this.

The problem of globally registered plugins is solved like this:

`ChartJS.plugins.clear();`

this must be done in everything always, because when it is done

```
import datalabels from 'chartjs-plugin-datalabels';
import deferred from 'chartjs-plugin-deferred';
```

All ChartJS already have that plugin registered.

Reply:
all global plugins can be used inline

`ChartJS.plugins.clear();`

is the way to keep ChartJS plugins isolated